CHANGELOG.md

```diff
@@ -9,8 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Update `dusk-poseidon` from `0.21` to `0.22.0-rc` [#59](https://github.com/dusk-network/schnorr/issues/59)
-- Update `dusk-pki` from `0.7` to `0.8.0-rc` [#59](https://github.com/dusk-network/schnorr/issues/59)
+- Update `dusk-pki` to `0.9` [#62](https://github.com/dusk-network/schnorr/issues/62)
+- Update `dusk-plonk` to `0.9` [#62](https://github.com/dusk-network/schnorr/issues/62)
+- Update `dusk-poseidon` to `0.23` [#62](https://github.com/dusk-network/schnorr/issues/62)
```

Comment:
Use the same format as below to be consistent:

- Update `lib` from `x.y.z` to `x.y.z`